dev-tools/kore-rpc-dev/Server.hs

```diff
@@ -240,6 +240,7 @@ main = do
                     server =
                         jsonRpcServer
                             srvSettings
+                            False -- no bound threads
                             (\rawReq -> runBoosterLogger . respond (koreRespond $ getReqId rawReq))
                             [Kore.handleDecidePredicateUnknown, handleErrorCall, handleSomeException]
                     interruptHandler _ = do
```

Comment:
Don't we still need the LLVM backend for simplification in `booster-dev`?

Reply:
There is a small test-suite in KEVM for `booster-dev` that we could try with this and updated KEVM. This one: https://github.com/runtimeverification/evm-semantics/blob/ed164b4aacd2b36b752a9eeebebc198f892cea28/.github/workflows/test-pr.yml#L91

Reply:
Oh I see, are we _always_ disabling bound threads here?

Reply:
This `kore-rpc-dev` is the legacy backend's RPC server with the CLI of `kore-rpc-booster` (same options and exposed features), purely for testing (not included in the K installation). It does not use the LLVM backend, so no point running with bound threads.
In `booster-dev` we have [this change](https://github.com/runtimeverification/haskell-backend/pull/4081/files#diff-a9c1dc73a4fe9f65f6831859e8c2a42a6ba08e46fdf8567a2a93117c9ca1ea51R166) to use bound threads when we have an LLVM library (same as in the `kore-rpc-booster` code, `booster/Server.hs`)